pac/source-repo/README.md

```diff
@@ -0,0 +1,22 @@
+# docker-build-ai-rhdh
+
+## Shared Git resolver model for shared pipeline and tasks
+ 
+This pipeline is used to create dockerfile based sscs builds. The pipeline run by this runner will clone the source, build an image with SBOM, and attestations and push these to the users image registry.  
```

Comment:
Not sure how much we can edit the content tho
```suggestion
This pipeline is used to create containerfile based SSCS (Software Supply Chain Security) builds. The pipeline run by this runner will clone the source, build an image with SBOM (Software Bill of Materials), and attestations and push these to the users image registry.  
```

Reply:
We can edit it as much as we want since as far as I understand we are diverging fully from rhtap so we own these definitions! I added your suggestions as part of https://github.com/redhat-ai-dev/rhdh-pipelines/pull/1/commits/7849f2ed4444ddc127f911786e37fd2dd3111e60